src/shared/logger/activation.ts

```diff
@@ -159,4 +159,30 @@ async function registerLoggerCommands(context: vscode.ExtensionContext): Promise
             recordVscodeViewLogs()
         })
     )
+    context.subscriptions.push(
+        vscode.commands.registerCommand('aws.viewLogsAtMessage', async (logID: number) => {
+            const msg: string | undefined = getLogger().getTrackedLog(logID, LOG_PATH)
+            await vscode.commands.executeCommand('aws.viewLogs')
+            const editor: vscode.TextEditor | undefined = vscode.window.activeTextEditor      
+
+            if (!msg || !editor) {
+                return
+            }
+
+            // Retrieve where the message starts by counting number of newlines
```

Comment:
There is no guarantee that this file isn't modified outside the IDE. Chances of it being changed are slim but they do exist (and it's more likely when you flash the log file in front of the user). If you want to try and prevent this from happening, you may want to use a [Virtual Document](https://code.visualstudio.com/api/extension-guides/virtual-documents), which will make the document readonly. Did something similar for the Cloudwatch Logs integration.

Reply:
Good idea! This would prevent the user from accidentally deleting the message as well.